### PR TITLE
Version bug fix

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: templates/node.zero-os/actions.py [hunk 1]
```diff
@@ -12,14 +12,17 @@ def get_statsdb(service):
...
 def get_version(job):
     from zeroos.orchestrator.sal.Node import Node
     from zeroos.orchestrator.configuration import get_jwt_token
     service = job.service
     node = Node.from_ays(service, get_jwt_token(job.service.aysrepo))
-    service.model.data.version = node.client.ping().split('Version:')[1]
+
+    pong = node.client.ping()
+    version = pong.split('Version:')[1] if pong else ''
+    service.model.data.version = version if version else service.model.data.version
```
Comment (zaibon) [inline templates/node.zero-os/actions.py]: I don't get this line

File: templates/node.zero-os/actions.py [hunk 2]
```diff
@@ -133,7 +137,7 @@ def monitor(job):
         try:
             node = Node.from_ays(service, token, timeout=5)
             node.client.testConnectionAttempts = 0
-            state = node.client.ping()
+            state = get_version(job)
```
Comment (zaibon) [inline templates/node.zero-os/actions.py]: not sure why get_version is assigned to a state variable.

Comment (abdulrahmantkhalifa) [reply templates/node.zero-os/actions.py]: it is the same as the pong it just returns a string , as far as i can tell it is just used as bool

Comment (zaibon) [reply templates/node.zero-os/actions.py]: This is not correct then. This should only do a ping. 
the `get_version` can return a not empty string even if the ping failed

**Pull request merged**